pkg/artifactory/provider/provider.go

```diff
@@ -1,4 +1,4 @@
 package provider
 
-var Version = "12.0.0" // needs to be exported so make file can update this
+var Version = "12.3.3" // needs to be exported so make file can update this
```

Comment:
This is automatically updated during build process by GoReleaser https://github.com/jfrog/terraform-provider-artifactory/blob/master/.goreleaser.yml#L16. So this is not necessary.